course05/homework01/martin/string.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <string>
+#include <ostream>
+#include <iostream>
+
+class String
+{
+public:
+	String();
+	String(std::string);
```

Comment:
This copies the string, which causes extra unnecessary work (alloc, copy). You should pass it  as const ref